pwa/src/components/common/multiDimensionalArrayInput.tsx

```diff
@@ -49,7 +51,7 @@ export function MultiDimensionalArrayInput(props: MultiDimensionalArrayInputProp
                       type="button"
                       className="utrecht-button utrecht-button-sm btn-sm btn-danger"
                     >
-                      Delete
+                      <FontAwesomeIcon icon={faTrash} /> Delete
```

Comment:
This one isn't actually working (at least not in the `AttributeForm`). This is because this button is actually rendered from `pwa/src/components/common/elementCreation.tsx`.

Reply:
<img width="431" alt="Schermafbeelding 2022-02-22 om 17 30 27" src="https://user-images.githubusercontent.com/55533449/155293794-6b35ef5f-a338-44f5-9533-84e1ed5272c6.png">